src/util.js

```diff
@@ -37,11 +37,7 @@ export const FUNCTION_CONVERT_TO_NUMBER_PARSE_ERROR =
  * @returns {boolean}
  */
 export function isNonNull(variable) {
-    if (variable == null || variable == undefined) {
-        return false;
-    } else {
-        return true;
-    }
+    return variable == null || variable == undefined ? false : true;
```

Comment:
Pretty sure you can simply this into `return variable != null`

Reply:
@danielakhterov great catch!

Reply:
If an empty string can be considered false-y then we can just reduce this to `!!variable`.